### PR TITLE
Sky 3 upgrade

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,14 +28,61 @@
     "@angular/core": ">=4.3.6",
     "@angular/forms": ">=4.3.6",
     "@angular/router": ">=4.3.6",
-    "@skyux/core": "^3.5.1",
-    "@skyux/i18n": "^3.5.0",
-    "@skyux/indicators": "^3.0.0"
+    "@skyux-sdk/builder": "^3.7.0",
```
Comment (Blackbaud-SteveBrush) [inline package.json]: Looks like the migration added some peers.

File: src/app/public/modules/flyout/flyout.component.spec.ts [hunk 2]
```diff
@@ -20,7 +20,8 @@ import {
 } from '@skyux-sdk/testing';
 
 import {
-  Observable
+  Observable,
+  throwError
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/flyout/flyout.component.spec.ts]: We can't migrate to RxJS 6 just yet, because it wouldn't work with SKY UX 2 SPAs.

Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/flyout/flyout.component.spec.ts]: I've reverted to the correct import for compatibility and have added a note to do the change later

File: package.json (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply package.json]: I fixed this :)

**Pull request merged**